core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -82,19 +85,20 @@ var ExplorationEditorSettingsTab = function() {
     await action.waitForAutosave();
     await action.click(
       'Disable Correctness Feedback Button', disableCorrectnessFeedbackButton);
-    await action.click('Neutral element', neutralElement);
+    await action.click('Neutral element', neutralElement, true);
```

Comment:
Just doing a migration shouldn't change whether an element is masked or not

Reply:
removed